src/photos/cameraTypes.ts

```diff
@@ -0,0 +1,11 @@
+export const enum cameraTypes {
```

Comment:
Created this because the task asked for it but I'm not sure where I should have used this. Maybe to check that the request was correct?